test/Microsoft.AspNetCore.Testing.Tests/OSSkipConditionTest.cs

```diff
@@ -32,7 +33,7 @@ public void RunTest_DoesNotRunOnWin7OrWin2008R2()
         {
             Assert.False(
                 RuntimeInformation.IsOSPlatform(OSPlatform.Windows) &&
-                Microsoft.Extensions.Internal.RuntimeEnvironment.OperatingSystemVersion.StartsWith("6.1"),
+                Environment.OSVersion.Version.ToString().StartsWith("6.1"),
```

Comment:
@blowdart would be proud. But it is a legit site. It got hit with the StartCom de-trust in Chrome. See https://github.com/terrajobst/apisof.net/issues/3